dialogue/TownNPCs/Mocha/InitialInteractions/Azura/Azura_Mocha_First3.json

```diff
@@ -30,7 +30,7 @@
     "pl": [
       "Serio? Dobrze słyszeć! To oznacza, że nie muszę się martwić o spadku rezerwacji.",
       "Brzmisz trochę niepewna... Na pewno wszystko dobrze ci jest, @CharacterName?",
```

Comment:
I think it would be better with just "Na pewno wszystko dobrze, @CharacterName?"